src/lib.rs

```diff
@@ -155,21 +171,22 @@ pub fn generate_commp_filecoin_proofs<R: Sized + io::Read>(
     })
 }
 
+/*
```

Comment:
Is there any reason to keep that code? Why not just removing it (and all references), we still have it in the git history.

Reply:
Yeah, good point, I meant to do a bit more with dead code cleaning here. But, the point of this library is twofold - making a Lambda and exploring the methods to make this happen with the various filecoin_proofs APIs. We only really need one of the codepaths in lib.rs, making 3 others redundant. But they're there for the commandline just to test, experiment and demonstrate them!

So I've reenabled this section after updating it, cleaned some dead code and added docs at the head of each of the main functions.